src/sas/qtgui/Calculators/UI/SldPanel.ui

```diff
@@ -338,6 +338,9 @@
         <property name="text">
          <string>Recalculate</string>
         </property>
+        <property name="autoDefault">
+         <bool>false</bool>
```

Comment:
Should we maybe set this to "True"? 

This way, hitting Recalculate will trigger the action, giving users quicker feedback (as opposed to moving hand to mouse, moving cursor to button, clicking).